docs/admin/views.mdx

```diff
@@ -93,6 +93,7 @@ For more granular control, pass a configuration object instead. Payload exposes
 | **`path`** \*      | Any valid URL path or array of paths that [`path-to-regexp`](https://www.npmjs.com/package/path-to-regex) understands.                                                                     |
 | **`exact`**        | Boolean. When true, will only match if the path matches the `usePathname()` exactly.                                                                                                     |
 | **`strict`**       | When true, a path that has a trailing slash will only match a `location.pathname` with a trailing slash. This has no effect when there are additional URL segments in the pathname. |
+| **`public`**       | When true, this path will be accessible to non logged in users allowing you to control access. Combine it with `exact: true` if you want nested paths to not be public. |
```

Comment:
Is this the right way or should we make an access function specific to the view?